src/plugins/intel_cpu/src/nodes/executors/reduce.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu_memory.h"
+#include "onednn/iml_type_mapper.h"
+#include "dnnl_scratch_pad.h"
+#include "executor.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+struct ReduceAttrs {
+    std::string nodeName;
```

Comment:
nodeName is definetelly not the part of operation attributes. Should be deleted.

Reply:
Deleted